configuration/configuration.go

```diff
@@ -313,6 +314,11 @@ func (c *Data) GetOpenshiftJenkinsVersion() string {
 	return c.v.GetString(varOpenshiftJenkinsVersion)
 }
 
+// GetOpenshiftJenkinsRootURL returns the URL of the Jenkins URL
+func (c *Data) GetOpenshiftJenkinsRootURL() string {
```

Comment:
No need for a Getter. Follow other 'pure' template vars: https://github.com/chmouel/fabric8-tenant/blob/0d6fadc590037b65564d7acddedb8daa728c5ce4/configuration/configuration.go#L366